data/zilliqa/assets/zil1hau7z6rjltvjc95pphwj57umdpvv0d6kh2t8zk/meta.json

```diff
@@ -3,13 +3,16 @@
   "symbol": "CARB",
   "web": "https://www.carbontoken.info",
   "supply": "10000000",
+  "email": "contact@carbontoken.info",
+  "whitepaper": "https://github.com/GenesysLabs/whitepapers"
```

Comment:
doesn't look like these are related to the carbon token, so you can remove it

Reply:
The whitepapers were written initally to be developed on ethereum. If you look at the website you will see that we are now building on Zilliqa under CARBON Token. https://www.carbontoken.info/projects/uto/ and https://www.carbontoken.info/projects/avatar/

Hopes that clarifies. thanks

Reply:
Gotcha, as long as these projects require CARBON it makes sense